layers/+lang/major-modes/packages.el

```diff
@@ -108,3 +109,6 @@
     :defer t
     :interpreter "\\(Wolfram\\|Mathematica\\)Script\\( -script\\)?"
     :mode "\\.wl\\'"))
+
+(defun major-modes/post-init-evil-matchit ()
+  (add-hook `matlab-mode-hook `turn-on-evil-matchit-mode))
```

Comment:
```suggestion
  (add-hook 'matlab-mode-hook 'turn-on-evil-matchit-mode))
```

Reply:
Thanks; the `'` seems more natural to the way I typically add hooks but I patterned this code on how the [`python` layer turns on `evil-matchit-mode`](https://github.com/syl20bnr/spacemacs/blob/3875f7a2fdef2f7373a1f01622dc228b1ed2b534/layers/%2Blang/python/packages.el#L145-L146). I'm perfectly happy to follow your suggestion, but curious if you know why [that code](https://github.com/syl20bnr/spacemacs/blob/3875f7a2fdef2f7373a1f01622dc228b1ed2b534/layers/%2Blang/python/packages.el#L145-L146) uses `` ` `` instead of `'`?

Reply:
`` ` `` is the backquote construct which allows quasi-quotation.
But when there's nothing no be quasi-quoted, `'` should be preferred.